lawlibrary/templates/peachjam/about.html

```diff
@@ -0,0 +1,67 @@
+{% extends "peachjam/layouts/main.html" %}
+{% load i18n %}
+
+{% block title %} {% trans 'About' %}{% endblock %}
+
+{% block page-content %}
+<section class="py-5">
+  <div class="container">
+    <h1 class="mb-4">{% trans 'About Lawlibrary.org.za' %}</h1>
+    <p class="lead mb-4">
+      {% blocktrans trimmed %}
+        Lawlibrary makes South African legal information freely accessible to everyone, in support of the rule of law,
+        access to justice, and innovation.
+      {% endblocktrans %}
+    </p>
+
+    <p class="mb-4">
+      {% blocktrans trimmed %}
+        Lawlibrary.org.za is curated by <a href="https://africanlii.org/" target="_blank" rel="noopener noreferrer">AfricanLII</a>
```

Comment:
I joined 
```
Lawlibrary.org.za is curated by [AfricanLII](https://africanlii.org/) in partnership with [JIFA](http://www.jifa.uct.ac.za/) and [Laws.Africa](https://laws.africa/).
```
and 
```
AfricanLII and Laws.Africa are members of the regional and global [Free Access to Law](http://www.fatlm.org/) community and subscribe to the principles of the Montreal Declaration on Free Access to Law.
```
to one paragraph, so it goes over two lines.
Otherwise, if they are separate paragraphs, it somewhat looks strange. (Too much open space used imo). But as always, open to go back to the original suggestion if you insist.